subprojects/blueprint-compiler.wrap

```diff
@@ -0,0 +1,8 @@
+[wrap-git]
+directory = blueprint-compiler
+url = https://gitlab.gnome.org/jwestman/blueprint-compiler.git
+revision = v0.4.0
+depth = 1
+
+[provide]
+program_names = blueprint-compiler
```

Comment:
Since the project uses Flatpak for development, there is no need to add a meson subproject for blueprint, just adding a blueprint module in flatpak manifest is enough. [docs](https://jwestman.pages.gitlab.gnome.org/blueprint-compiler/flatpak.html)

Reply:
Yes, but it's good to keep it for others distribution way ?

Reply:
That is a fair point but all most common distros (Debian, Ubuntu, Fedora and Arch Linux) have blueprint-compiler already in the repos. If some don't, that might be a problem (it's unlikely tho since most distros that have up to date GTK4 and libadwaita have blueprint-compiler too) but I'd prefer not to add any more stuff to the repo than needed. Since it's most likely not needed I think it's okay to not add it.